cn.lpkt.serverbox.app.desktop

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Name=ServerBox
+Exec=/app/AppRun
+Icon=io.github.serverbox.png
```

Comment:
this icon name is incorrect. It doesn't match the app-id and should never include the file extension (icon name not icon file name)